packages/web/integrations/notifi/notifi-popover.tsx

```diff
@@ -0,0 +1,128 @@
+import classNames from "classnames";
+import React, { ComponentProps, Fragment, FunctionComponent } from "react";
+
+import { useStore } from "~/stores";
+
+import { Icon } from "../../components/assets";
+import { Button } from "../../components/buttons";
+import IconButton from "../../components/buttons/icon-button";
+import { Popover } from "../../components/popover";
+import { useNotifiModalContext } from "./notifi-modal-context";
+import { NotifiSubscriptionCard } from "./notifi-subscription-card";
+
+export interface NotifiButtonProps {
+  className?: string;
+  hasUnreadNotification: boolean;
+}
+
+const NotifiIconButton: FunctionComponent<
+  ComponentProps<typeof Button> & { hasUnreadNotification?: boolean }
+> = (props) => {
+  return (
+    <>
+      <IconButton
+        aria-label="Open Notifications dropdown"
+        icon={<Icon id="bell" width={24} height={24} />}
+        {...props}
+      />
+      {props.hasUnreadNotification ? (
+        <div className="absolute bottom-[4.6px] right-[4.85px]">
+          <svg
+            width="10"
+            height="10"
+            viewBox="0 0 10 10"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <ellipse
+              cx="5.08333"
+              cy="4.99998"
+              rx="4.33333"
+              ry="4.4"
+              fill="#FA825D"
+            />
+          </svg>
+        </div>
+      ) : null}
+    </>
+  );
+};
+
+export const NotifiPopover: FunctionComponent<NotifiButtonProps> = ({
+  className,
+  hasUnreadNotification,
+}: NotifiButtonProps) => {
+  const {
+    chainStore: {
+      osmosis: { chainId },
+    },
+    accountStore,
+  } = useStore();
+
+  const {
+    innerState: { onRequestBack, backIcon, title } = {},
+    isOverLayEnabled,
+    setIsOverLayEnabled,
+  } = useNotifiModalContext();
+
+  if (!accountStore.getWallet(chainId)) {
+    return <NotifiIconButton className={className} disabled />;
+  }
+
+  return (
+    <>
+      {isOverLayEnabled && (
+        <div
+          className="fixed left-0 top-0 bottom-0 right-0 z-[1] bg-osmoverse-1000 opacity-90"
+          onClick={() => setIsOverLayEnabled(false)}
+        ></div>
+      )}
+      <Popover className="relative z-[1]">
+        <Popover.Button as={Fragment}>
+          <NotifiIconButton
+            className={className}
+            hasUnreadNotification={hasUnreadNotification}
+          />
+        </Popover.Button>
+        <Popover.Panel
+          className={classNames(
+            "absolute bottom-[-0.5rem] right-[-10rem] z-40",
+            `h-[640px] w-[420px]`,
```

Comment:
How does this work on mobile? Has mobile been considered?

Reply:
Yeah, Mobile is also considered. 
The `NotifiPopover`  container is used for desktop view. 
The mobile view adopts `NotifiModal` is used for for small view